src/main/java/de/unistuttgart/memorybackend/data/OverworldResultDTO.java

```diff
@@ -26,7 +26,7 @@ public class OverworldResultDTO {
      * The name of the minigame. In this case "MEMORY".
      */
     @NotNull(message = "game cannot be null")
-    final String game = "MEMORY";
+    static final String game = "MEMORY";
```

Comment:
```suggestion
    final String game = "MEMORY";
```
This can not be static. The `game` attribute is part of the OverworldResultDTO and the overworld backend is expecting this in the request object as an attribute.